.murdock

```diff
@@ -240,37 +240,46 @@ compile() {
     [ $# -ne 2 ] && error "$0: compile: invalid parameters (expected \$appdir \$board:\$toolchain)"
     [ ! -d "$appdir" ] && error "$0: compile: error: application directory \"$appdir\" doesn't exist"
 
-    # compile
+    # We compile a first time with Kconfig based dependency
+    # resolution for regression purposes. $TEST_KCONFIG contains a
+    # list of board-application tuples that are currently modeled to
+    # run with Kconfig
+
+    # we can't use '-' in variable names
+    _board=$(echo ${board} | tr '-' '_')
```

Comment:
I don't understand why this is needed. I don't see where the content of ${board} is used as variable name.

Reply:
I think you are right, probably left over from the first idea which was to access via some sort of dictionary to the list, good catch. I'll try to change this.

Reply:
Removed